Wrap long column names in chart settings sidebar

shaunlebron opened the pull request:
Closes https://github.com/metabase/metabase/issues/32964

### Description

Fix the ChartSettingField width to wrap the column name to multiple lines. 

<table>
<tr><td>before</td><td>after</td></tr>
<tr><td valign="top"><img width="380" src="https://github.com/metabase/metabase/assets/116838/f84edc2c-4853-4236-85ff-a90de018460d"></td><td><img width="356" alt="CleanShot 2023-08-09 at 14 28 17@2x" src="https://github.com/metabase/metabase/assets/116838/63a37ed6-3b37-4051-aaf8-ae2ba843c459"></td></tr>
</table>



### How to verify

Describe the steps to verify that the changes are working as expected.

1. Open this question: http://localhost:3000/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjoxLCJ0eXBlIjoicXVlcnkiLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjIsImV4cHJlc3Npb25zIjp7IkEgdmVyeSBsb25nIGNvbHVtbiBuYW1lIHRoYXQgd2lsbCBjYXVzZSB0ZXh0IG92ZXJmbG93IjpbIioiLFsiZmllbGQiLDE3LG51bGxdLDJdfSwiYWdncmVnYXRpb24iOltbInN1bSIsWyJleHByZXNzaW9uIiwiQSB2ZXJ5IGxvbmcgY29sdW1uIG5hbWUgdGhhdCB3aWxsIGNhdXNlIHRleHQgb3ZlcmZsb3ciXV1dLCJicmVha291dCI6W1siZmllbGQiLDE0LHsiYmFzZS10eXBlIjoidHlwZS9EYXRlVGltZSIsInRlbXBvcmFsLXVuaXQiOiJ3ZWVrIn1dXX19LCJkaXNwbGF5IjoibGluZSIsInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnt9fQ==
4. Click the gear icon for visualization settings
5. Verify that the column name is wrapped to multiple lines.

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

shaunlebron committed:
limit the chart settings sidebar content width to prevent overflow

shaunlebron committed:
add e2e test

alxnddr reviewed (commented):
It would be nice to keep the behavior consistent with the table columns where the wrapping happens:
<img width="328" alt="Screen Shot 2023-08-08 at 8 33 10 PM" src="https://github.com/metabase/metabase/assets/14301985/656bc095-b4e1-4443-bf49-1d50751691be">

shaunlebron committed:
Wrap instead of ellipsify

shaunlebron committed:
prevent column handle from shrinking when column name wraps to multiple lines

shaunlebron commented:
> It would be nice to keep the behavior consistent with the table columns where the wrapping happens: <img alt="Screen Shot 2023-08-08 at 8 33 10 PM" width="328" src="https://user-images.githubusercontent.com/14301985/259258036-656bc095-b4e1-4443-bf49-1d50751691be.png">

Implemented, and updated the PR screenshots.  Also fixed the table column handle.  Thanks

<img width="315" alt="CleanShot 2023-08-09 at 14 40 05@2x" src="https://github.com/metabase/metabase/assets/116838/756982dc-764a-4acc-8b79-4102fa859446">

shaunlebron committed:
fix e2e spec
